--- configure.orig	2018-06-18 20:58:58.614591131 +0000
+++ configure.ac	2018-06-18 20:59:03.000000000 +0000
@@ -574,6 +574,8 @@
 AST_EXT_LIB_SETUP([X11], [X11], [x11])
 AST_EXT_LIB_SETUP([ZLIB], [zlib compression], [z])
 
+AST_EXT_LIB_SETUP([DIALOGFLOW], [Google Dialogflow], [dialogflow])
+
 # check for basic system features and functionality before
 # checking for package libraries
 
@@ -2525,6 +2527,8 @@
 AST_EXT_LIB_CHECK([VORBIS], [vorbis], [vorbis_info_init], [vorbis/codec.h], [-lm -lvorbisenc -lvorbisfile])
 AST_C_DECLARE_CHECK([VORBIS_OPEN_CALLBACKS], [OV_CALLBACKS_NOCLOSE], [vorbis/vorbisfile.h])
 
+AST_EXT_LIB_CHECK([DIALOGFLOW], [dialogflow], [df_init], [libdialogflow.h], [-lprotobuf -lgrpc++ -lstdc++])
+
 AC_LANG_PUSH(C++)
 
 if test "${USE_VPB}" != "no"; then
