--- configure.ac
+++ configure.ac
@@ -2118,6 +2118,10 @@ else
 fi
 AST_C_DECLARE_CHECK([VORBIS_OPEN_CALLBACKS], [OV_CALLBACKS_NOCLOSE], [vorbis/vorbisfile.h])
 
+# BEGIN RES_SPEECH_DFE
+AST_EXT_LIB_SETUP([DFEGRPC], [Dialogflow gRPC], [dfegrpc])
+AST_EXT_LIB_CHECK([DFEGRPC], [dfegrpc], [df_init], [libdfegrpc.h], [-lprotobuf -lgrpc++ -lstdc++])
+# END RES_SPEECH_DFE
 AC_LANG_PUSH(C++)
 
 if test "${USE_VPB}" != "no"; then
